app/helpers/ui_helper.rb

```diff
@@ -110,8 +110,11 @@ def edit_button_to(link, options = {}, properties = {})
     _link_to link, { icon: "pencil-square-o", type: "primary", text: "Edit", size: "xs" }.merge(options), properties
   end
 
-  def filter_button(options = {})
-    _button_to({ icon: "filter", type: "primary", text: "Filter", size: "md" }.merge(options))
+  def filter_button(options = {}, data = {})
+    _button_to(
+      { icon: "filter", type: "primary", text: "Filter", size: "md" }.merge(options),
+      { data: { test_id: "filter-button" }.merge(data) }
```

Comment:
Was this added by accident? I don't see it used.

Reply:
This adds a data-test-id to the filter button which is used in the shared example system spec:
```ruby
page.execute_script("document.querySelector('[data-test-id=\"filter-button\"]').click();")
```

Normally would simply use the Capybara click method with text "Filter", but due to event handling required to reproduce this issue, had to use execute_script with querySelector. And there wasn't an easy way to select a button by its text. So this was to ensure that the test is clicking the correct button, in case there should be more buttons added to any of the pages in the future.

Reply:
Actually, after working through the CI failing tests, this is no longer needed, removed.